bash_scripting.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Check if the parameter is provided, otherwise use the default value
+if [ -z "$1" ]; then
```

Comment:
This works. You can also you a one-liner for default arguments like this: 
`NUMBER=${1:-50}`